lambdas/service/app.py

```diff
@@ -831,36 +836,101 @@ def get_order():
-        - name: format
-          in: query
-          type: string
-          description: The desired format of the output. Possible values are `compact` (the default) for a tab-separated
-          manifest and `terra.bdbag` for a manifest in the format documented `http://bd2k.ini.usc.edu/tools/bdbag/. The
-          latter is essentially a ZIP file containing two manifests: one for participants (aka Donors) and one for
-          samples (aka specimens). The format of the manifests inside the BDBag is documented here:
-          https://software.broadinstitute.org/firecloud/documentation/article?id=10954
-        - name: token
-          in: query
-          type: string
-          description: Reserved. Do not pass explicitly.
+                - `compact` (the default) for a compact tab-separated manifest
 
-    :return: If the manifest generation has been started or is still ongoing, the response will have a
-    301 status and will redirect to a URL that will get a recheck the status of the manifest.
+                - `full` for a full tab-separated manifest
 
-    If the manifest generation is done and the manifest is ready to be downloaded, the response will
-    have a 302 status and will redirect to the URL of the manifest.
-    """
+                - `terra.bdbag` for a manifest in the
+                  [bdbag format](http://bd2k.ini.usc.edu/tools/bdbag/). This
+                  provides a ZIP file containing two manifests: one for
+                  Participants (aka Donors) and one for Samples (aka Specimens).
+                  For more on the format of the manifests see
+                  [documentation here](https://software.broadinstitute.org/firecloud/documentation/article?id=10954).
+            ''',
+        ),
+        params.query(
+            'token',
+            schema.optional({'type': 'string'}),
+            description='Reserved. Do not pass explicitly.',
+        ),
+    ],
+}
+
+
+# Copy of path_spec required due to FIXME: https://github.com/DataBiosphere/azul/issues/1646
+@app.route('/manifest/files', methods=['GET'], cors=True, path_spec=copy.copy(manifest_path_spec), method_spec={
+    'tags': ['Manifests'],
+    'summary': 'Request a download link to a manifest file and redirect',
+    'description': format_description('''
+        Initiate and check status of a manifest generation job, returning
+        either a 301 response redirecting to a URL to re-check the status of
+        the manifest generation or a 302 response redirecting to the location
+        of the completed manifest.
+    '''),
+    'responses': {
+        '301': {
+            'description': format_description('''
+                The manifest generation has been started or is still ongoing.
+                The response is a redirect back to this endpoint, so the client
+                should expect a subsequent response of the same kind.
+            '''),
+            **{
```

Comment:
Is the `**` necessary?

Reply:
Good catch. It was necessary when the `**` was unpacking a dictionary stored in a variable, but certainly not now. My fault for not checking PyCharm's refactor inline results more closely.